stable/coscale/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for CoScale Helm package.
+
+image:
+    repository: "coscale/coscale-agent"
+    tag: "latest"
+    pullPolicy: "IfNotPresent"
```

Comment:
It would be better to set the tag to a version (e.g. 3.7.0). If you need to use latest, the `pullPolicy` should be Always.